e2e_samples/unstructured_data/src/common/citation_db.py

```diff
@@ -17,11 +15,7 @@
 
 
 def get_conn(conn_str: str) -> pyodbc.Connection:
-    credential = DefaultAzureCredential()
```

Comment:
Instead of removing functionality, can you add some kind of switch to use either default azure credential or the connection string? We might end up switching back to that and would be nice to support both. An env variable can control this